kie-wb-common-forms/kie-wb-common-dynamic-forms/kie-wb-common-dynamic-forms-client/src/main/java/org/kie/workbench/common/forms/dynamic/client/rendering/renderers/date/input/DatePickerWrapper.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.workbench.common.forms.dynamic.client.rendering.renderers.date.input;
+
+import java.util.Date;
+
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import com.google.gwt.event.logical.shared.ValueChangeEvent;
+import com.google.gwt.event.logical.shared.ValueChangeHandler;
+import com.google.gwt.event.shared.GwtEvent;
+import com.google.gwt.event.shared.HandlerRegistration;
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+
+@Dependent
+public class DatePickerWrapper implements IsWidget, DatePickerWrapperView.Presenter {
+
+    private DatePickerWrapperView view;
+
+    @Inject
+    public DatePickerWrapper(DatePickerWrapperView view) {
+        this.view = view;
+        view.setPresenter(this);
+    }
+
+    @Override
+    public Widget asWidget() {
+        return view.asWidget();
+    }
+
+    @Override
+    public void setValue(Date values) {
+        setValue(values,false);
+    }
+
+    @Override
+    public void setValue(Date value,
+                         boolean fireEvents) {
+        view.setDateValue(value);
+        if (fireEvents) {
+            ValueChangeEvent.fire(this, value);
+        }
+    }
+
+    @Override
+    public void setId(String id) {
+         view.setId(id);
+    }
+
+    @Override
+    public void setName(String name) {
+       view.setName(name);
+    }
+
+    @Override
+    public void setPlaceholder(String placeholder) {
+       view.setPlaceholder(placeholder);
+    }
+
+    @Override
+    public void setEnabled(Boolean enabled) {
```

Comment:
```suggestion
    public void setEnabled(boolean enabled) {
```

I would rather use primitive type of boolean.